.gitignore

```diff
@@ -1,4 +1,6 @@
 MMOEngine
 luacov.report.out
 luacov.stats.out
-MMOCoreORB/bin/conf/config.lua
\ No newline at end of file
+MMOCoreORB/bin/conf/config.lua
+MMOCoreORB/bin/scripts/managers/ghoutput.xml
+MMOCoreORB/bin/scripts/managers/intendedspawns.txt
```

Comment:
Can we go ahead and add resource_manager_spawns.lua to gitignore as well? It's a pain to deal with